lib/readline.js

```diff
@@ -114,6 +118,11 @@ function Interface(input, output, completer, terminal) {
     completer = input.completer;
     terminal = input.terminal;
     historySize = input.historySize;
+    if (input.tabSize !== undefined) {
+      const positive = true;
+      validateUint32(input.tabSize, 'tabSize', positive);
```

Comment:
Nit: We can directly pass `true` in third argument instead of creating a new variable and passing it

Reply:
I think this was intentional to make the argument passed descriptive instead of just being `true`.